packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue

```diff
@@ -60,30 +61,10 @@
 						{{ $locale.baseText('nodeCreator.noResults.or') }}
 					</template>
 
-					<no-results v-else :showRequest="filteredAllNodeTypes.length === 0" :show-icon="filteredAllNodeTypes.length === 0">
-						<!-- There are results in other sub-categories/tabs  -->
-						<template #title>
-							<p v-html="$locale.baseText('nodeCreator.noResults.clickToSeeResults')" />
-							<p v-if="filteredAllNodeTypes.length === 0" v-text="$locale.baseText('nodeCreator.noResults.weDidntMakeThatYet')" />
-							<p v-else v-html="$locale.baseText('nodeCreator.noResults.clickToSeeResults')" />
-						</template>
-
-						<!-- Regular Search -->
-						<template v-if="filteredAllNodeTypes.length === 0" #action>
-							{{ $locale.baseText('nodeCreator.noResults.dontWorryYouCanProbablyDoItWithThe') }}
-							<n8n-link @click="selectHttpRequest" v-if="[REGULAR_NODE_FILTER, ALL_NODE_FILTER].includes(selectedType)">
-								{{ $locale.baseText('nodeCreator.noResults.httpRequest') }}
-							</n8n-link>
-							<template v-if="selectedType === ALL_NODE_FILTER">
-								{{ $locale.baseText('nodeCreator.noResults.or') }}
-							</template>
-
-							<n8n-link @click="selectWebhook" v-if="[TRIGGER_NODE_FILTER, ALL_NODE_FILTER].includes(selectedType)">
-								{{ $locale.baseText('nodeCreator.noResults.webhook') }}
-							</n8n-link>
-							{{ $locale.baseText('nodeCreator.noResults.node') }}
-						</template>
-					</no-results>
+					<n8n-link @click="selectWebhook" v-if="[TRIGGER_NODE_FILTER, ALL_NODE_FILTER].includes(selectedType)">
```

Comment:
why are we changing this here?

Reply:
Fair point, I've extracted this change to [its own PR](https://github.com/n8n-io/n8n/pull/4721).

Reply:
no worries. just was not sure if intentional or not.